app/task_routes.py

```diff
@@ -0,0 +1,103 @@
+import datetime
+import requests
+from flask import Blueprint, jsonify, request, abort, make_response
+from app.models.task import Task
+from app import db
+from app.models.goal import Goal
+from sqlalchemy import desc
+from sqlalchemy import asc
+import os
+
+tasks_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def handle_tasks():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return jsonify(details="Invalid data"), 400
+
+        new_task = Task(
+            title=request_body["title"],
+            description=request_body["description"],
+            completed_at=request_body["completed_at"]
+        )
+
+        db.session.add(new_task)
+        db.session.commit()
+
+        return jsonify(task=new_task.to_json()), 201
+
+    elif request.method == "GET":
+        query = request.args.get("sort")
+        if query == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+        elif query == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+        else:
+            tasks = Task.query.all()
+
+        response_body = []
+        for task in tasks:
+            response_body.append(task.to_json())
+
+        return jsonify(response_body), 200
```

Comment:
You _do_ have to call `jsonify` here, though, as `response_body` is a list, not a dictionary.

However, when the status code is 200, you don't have to actually supply it, because Flask will use that code by default, so you could replace this `return` statement with `return jsonify(response_body)`.

However, including the status code can be easily justified as a way to be more explicit to help a reader, and you've been pretty consistent about including that, so it's likely fine!